### PR TITLE
Why python?

### PR DESCRIPTION
**Pull request by jukent**
I tried to split the intro content into smaller deliverables to iterate over and publish. Here is the PR for what I envision is one of the first things we cover in teaching Python - why they should learn Python and how it differs from other languages.

### CODE REVIEW

**Comment by jukent** [comment]
Hey @ProjectPythia/education did you get an email when the whole team was requested for review? One goal was to have this merged by tomorrow so I just requested review from individuals.

**Comment by dcamron** [comment]
I do believe I received an e-mail when you requested review to the entire education team initially. I'll get my review in today!

File: foundations/why-python.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# Why Python?
+
+You're already here because you want to learn to use Python for your data analysis and visualizations.
+
+Perhaps the #1 reason to use Python is because it is so widely used in the scientific community!
```
Comment (dcamron) [inline foundations/why-python.md:5]: Longer term we might even consider turning this line into something visually punchy, as it is the key point to sell to people! Maybe even just to a embedded image of this as a quote or something, not sure.

**Review by dcamron — APPROVED** [review]
I think the current body of text is punchy and highlights the important bits! I have no critiques about what's written.

A larger consideration for what's unwritten might be for us to tie the "#1 reason is that it is used by the community" and the "open source" sections to our focus on the established community of high level community python packages that using python gives access to. Similarly we could tie this "#1 reason" to "everyone else is using it so you can easily share your work and code and know what is happening". I think there's room to expand these here now or in more in-depth sections later, so I'm good to approve unless you think this feedback is important to incorporate into this page now.

File: foundations/why-python.md (hunk 1 above)
Comment (jukent) [reply foundations/why-python.md]: Just bolded this! We might want a different solution later.

File: foundations/why-python.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+## Open source
+
+Python as a language is open source which means that there is a community of developers behind its codebase. Anyone can join the developer community and contribute to deciding the future of the language. When someone identifies gaps to Python's abilities, they can write up the code to fill these gaps. The open source nature of Python means that Python as a language is very adaptable to shifting needs of the user community. This harks back to the idea that the wide-use of Python within the scientific community is a benefit to you! The large Python user base within your field has established high level community Python packages that are available to you in your workflow.
```
Comment (jukent) [inline foundations/why-python.md:25]: @dcamron I just changed this paragraph to respond to your comment. What do you think?

Comment (dcamron) [reply foundations/why-python.md:25]: I like it! Thanks @jukent. I still look forward to others' feedback, but with this I'm good

**Review by clyne — APPROVED** [review]
Looks great!

**Comment by brian-rose** [comment]
I just made two small changes to the text. Consider these suggestions but I thought it would speed things up to push them directly to this branch. Overall I think it's great!

**Comment by brian-rose** [comment]
I thought it was useful to list Fortran among the examples of "lower level languages" since many in the geoscience community are more familiar with Fortran than C.

**Review by brian-rose — APPROVED** [review]

**Comment by brian-rose** [comment]
@jukent if you're happy with my small edits above, I suggest that we merge now to stay on track. We can always keep tweaking in future PRs.

**Pull request merged**